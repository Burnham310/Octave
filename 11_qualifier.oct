main = { ;tonic=D, octave=5, mode=major, bpm=140, tempo=8/8;
    1'/4 ^^2'/4 ^7'/4 ^5'/4
    2 3 5 6 ^2 ^#4 ^5'/4

    1'/4 ^^2'/4 ^7'/4 ^5'/4
    2 3 5 6 ^2 ^#4 ^5 ^^2

    1'/4 ^^2'/4 ^7'/4 ^5'/4
    2 3 5 6 ^2 ^#4 ^5'/4
    
    1'/4 ^^2'/4 ^7'/4 [] 1 2
    3 5 6 7 ^2 ^5 ^6 ^5

    5'/4 ^2'/4 ^7'/4 ^[6 ^#4] ^[7 ^5]
    [] ^^[2 1] ^^3 ^^2 ^5 ^5 ^7 ^[5 7]

    1'/4 ^^2'/4 ^5'/2
    ^6 ^7 ^5 ^^#4 ^^5 ^^#4 ^5 ^[7 ^2]

    5'/4 ^^2'/4 ^7 ^^1'/4 ^^5
    [] ^^2 ^^3 ^^5 ^^6 ^^7 ^[5 7 ^2] ^[1 3]

    [] ^^2 ^^5 ^^7'/4 ^5'/4 []
    ^1 ^6 ^7 ^5 ^^2'/4 ^^5 ^^2
}
