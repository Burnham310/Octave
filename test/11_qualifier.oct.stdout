[
{
  "freq": 146.82925415039062,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 659.2410278320312,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 554.3533935546875,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 439.9906311035156,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 164.81027221679688,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 184.99327087402344,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 219.9953155517578,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 246.93638610839844,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 329.62054443359375,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 415.29583740234375,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 439.9906311035156,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 146.82925415039062,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 659.2410278320312,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 554.3533935546875,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 439.9906311035156,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 164.81027221679688,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 184.99327087402344,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 219.9953155517578,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 246.93638610839844,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 329.62054443359375,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 415.29583740234375,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 439.9906311035156,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 659.2410278320312,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 146.82925415039062,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 659.2410278320312,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 554.3533935546875,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 439.9906311035156,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 164.81027221679688,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 184.99327087402344,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 219.9953155517578,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 246.93638610839844,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 329.62054443359375,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 415.29583740234375,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 439.9906311035156,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 146.82925415039062,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 659.2410278320312,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 554.3533935546875,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0.000000000000000000000012766289857068152,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 146.82925415039062,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 164.81027221679688,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 184.99327087402344,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 219.9953155517578,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 246.93638610839844,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 277.1767272949219,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 329.62054443359375,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 439.9906311035156,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 493.8727722167969,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 439.9906311035156,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 219.9953155517578,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 329.62054443359375,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 554.3533935546875,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 493.8727722167969,
  "duration": 0.2142857164144516,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 830.5917358398438,
  "duration": 0.2142857164144516,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0.000000000000000000000025532579714136304,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 554.3533935546875,
  "duration": 0.2142857164144516,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 879.9812622070312,
  "duration": 0.2142857164144516,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0.000000000000000000000025532579714136304,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0.000000000000000000000012766289857068152,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 659.2410278320312,
  "duration": 0.2142857164144516,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 587.3170776367188,
  "duration": 0.2142857164144516,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0.00000000000000000000005106515942827261,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 739.9730834960938,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 659.2410278320312,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 439.9906311035156,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 439.9906311035156,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 554.3533935546875,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 439.9906311035156,
  "duration": 0.2142857164144516,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 554.3533935546875,
  "duration": 0.2142857164144516,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0.000000000000000000000025532579714136304,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 146.82925415039062,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 659.2410278320312,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 439.9906311035156,
  "duration": 0.8571428656578064,
  "gap": 0.8571428656578064,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 493.8727722167969,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 554.3533935546875,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 439.9906311035156,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 830.5917358398438,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 879.9812622070312,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 830.5917358398438,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 439.9906311035156,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 554.3533935546875,
  "duration": 0.2142857164144516,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 659.2410278320312,
  "duration": 0.2142857164144516,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0.000000000000000000000025532579714136304,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 219.9953155517578,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 659.2410278320312,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 554.3533935546875,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 587.3170776367188,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 879.9812622070312,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0.000000000000000000000012766289857068152,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 659.2410278320312,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 739.9730834960938,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 879.9812622070312,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 987.74560546875,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 1108.706787109375,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 439.9906311035156,
  "duration": 0.2142857164144516,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 554.3533935546875,
  "duration": 0.2142857164144516,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 659.2410278320312,
  "duration": 0.2142857164144516,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0.000000000000000000000025532579714136304,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 293.65850830078125,
  "duration": 0.2142857164144516,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 369.9865417480469,
  "duration": 0.2142857164144516,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0.000000000000000000000025532579714136304,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0.000000000000000000000012766289857068152,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 659.2410278320312,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 879.9812622070312,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 1108.706787109375,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 439.9906311035156,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0.000000000000000000000012766289857068152,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 293.65850830078125,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 493.8727722167969,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 554.3533935546875,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 439.9906311035156,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 659.2410278320312,
  "duration": 0.4285714328289032,
  "gap": 0.4285714328289032,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 879.9812622070312,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 659.2410278320312,
  "duration": 0.2142857164144516,
  "gap": 0.2142857164144516,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0,
  "duration": 0,
  "gap": 0,
  "amp": 0,
  "instrument": "dummy"
}

]